src/components/DecodeTxs/index.tsx

```diff
@@ -0,0 +1,200 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components'
+import { Transaction } from '@gnosis.pm/safe-apps-sdk-v1'
+import { Text, EthHashInfo, CopyToClipboardBtn, IconText, FixedIcon } from '@gnosis.pm/safe-react-components'
+import get from 'lodash.get'
+
+import { web3ReadOnly as web3 } from 'src/logic/wallets/getWeb3'
+import { getExplorerInfo } from 'src/config'
+import { DecodedData, DecodedDataBasicParameter, DecodedDataParameterValue } from 'src/types/transactions/decode.d'
+import { DecodedTxDetail } from 'src/routes/safe/components/Apps/components/ConfirmTxModal'
+
+const FlexWrapper = styled.div<{ margin: number }>`
+  display: flex;
+  align-items: center;
+
+  > :nth-child(2) {
+    margin-left: ${({ margin }) => margin}px;
+  }
+`
+
+const BasicTxInfoWrapper = styled.div`
+  margin-bottom: 15px;
+
+  > :nth-child(2) {
+    margin-bottom: 15px;
+  }
+`
+
+const TxList = styled.div`
+  width: 100%;
+  max-height: 260px;
+  overflow-y: auto;
+  border-top: 2px solid ${({ theme }) => theme.colors.separator};
+`
+
+const TxListItem = styled.div`
+  display: flex;
+  justify-content: space-between;
+
+  padding: 0 24px;
+  height: 50px;
+  border-bottom: 2px solid ${({ theme }) => theme.colors.separator};
+
+  :hover {
+    cursor: pointer;
+  }
+`
+const ElementWrapper = styled.div`
+  margin-bottom: 15px;
+`
+
+export const BasicTxInfo = ({
+  txRecipient,
+  txData,
+  txValue,
+}: {
+  txRecipient: string
+  txData: string
+  txValue: string
+}): ReactElement => {
+  return (
+    <BasicTxInfoWrapper>
+      {/* TO */}
+      <>
+        <Text size="lg" strong>
+          {`Send ${txValue} ETH to:`}
+        </Text>
+        <EthHashInfo
+          hash={txRecipient}
+          showIdenticon
+          textSize="lg"
+          showCopyBtn
+          explorerUrl={getExplorerInfo(txRecipient)}
+        />
+      </>
+      <>
+        {/* Data */}
```

Comment:
why these fragments? they don't seem to be required as we're under `<BasicTxInfoWrapper>`

Reply:
we need styles for the second child.

Reply:
but, the fragment translates into a DOM element?